src/CommunityToolkit.Maui.Core/Interfaces/IDockLayoutManager.shared.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.Maui.Layouts;
+
+namespace CommunityToolkit.Maui.Core.Interfaces;
+
+/// <summary>LayoutManager for <see cref="IDockLayout"/>.</summary>
+public interface IDockLayoutManager : ILayoutManager
```

Comment:
What's the value to have an empty interface?

Reply:
Originally, I introduced that interface to ensure that you can't inject any (potentially incompatible) ILayoutManager. For that, a more specific (but empty) interface IDockLayoutManager works well. However, I just checked the latest MAUI sources and they do not take any precautions in that direction either. Removing the interface makes sense then, I guess.